packages/dashboard/src/vaadin-dashboard.js

```diff
@@ -221,7 +221,11 @@ class Dashboard extends DashboardLayoutMixin(ElementMixin(ThemableMixin(PolylitM
 
   /** @protected */
   render() {
-    return html`<div id="grid"><slot></slot></div>`;
+    return html`<div id="grid">
+      <div id="slots" style="display: contents">
+        ${(this.items || []).map((_item, index) => html`<slot name="slot-${index}"></slot>`)}
```

Comment:
**perf:** The component currently unnecessarily re-renders whenever a section is moved, despite the slot order staying the same. Using a `__widgetCount` property instead of `items` would optimize this.

Reply:
Done